src/renderer/components/+user-management/+service-accounts/view.tsx

```diff
@@ -72,13 +69,3 @@ export class ServiceAccounts extends React.Component<Props> {
   }
 }
 
```

Comment:
Note: This component was extracted to separate file to avoid circular dependency